src/components/interfaces/HMI_API.xml

```diff
@@ -1569,6 +1649,32 @@
   </param>
 </struct>
 
+<struct name="VideoStreamingFormat">
+  <description>Video streaming formats and their specifications.</description>
+  <param name="protocol" type="Common.VideoStreamingProtocol" mandatory="true">
+    <description>Protocol type, see VideoStreamingProtocol</description>
+  </param>
+  <param name="codec" type="Common.VideoStreamingCodec" mandatory="true">
+    <description>Codec type, see VideoStreamingCodec</description>
+  </param>
+</struct>
+
+<struct name="VideoConfig">
+  <description>Configuration of a video stream.</description>
+  <param name="protocol" type="Common.VideoStreamingProtocol" mandatory="false">
+    <description>The video protocol configuration</description>
+  </param>
+  <param name="codec" type="Common.VideoStreamingCodec" mandatory="false">
+    <description>The video codec configuration</description>
+  </param>
```

Comment:
Alright, fair enough, I agree that we probably shouldn't change the spec for the feature without any discussion with the SDLC, the way it was done just seemed odd to me.